interview-questions.md

```diff
@@ -8,24 +8,38 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+# Your answer:
+The foreign key is the mapping on the Student model which indicates to which Cohort the student belongs to.
 
-Researched answer:
+# Researched answer:
+Because the foreign key is in the references column of the Student database, a migration to remove the reference and replace it with a correct column or to fix the foreign_key attribute may do the trick.
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+# Your answer:
+I do not know.
```

Comment:
I do not know is not an answer that will help you get hired in an interview.  Talk about concepts that you do know or show interest in the concept.